en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/subscription-validation.md

```diff
@@ -0,0 +1,28 @@
+# Subscription Validation Model
+
+When using WSO2 API Manager as the key manager with Choreo Connect, it can be configured to validate the API Subscriptions. For this, the same API should be published in both API Manager and Choreo Connect, and a valid access token (JWT or Reference token) should be obtained by subscribing to the API via an Application. Choreo Connect is capable of validating subscriptions only for the configured tenant (One tenant per Choreo Connect basis).
```

Comment:
```suggestion
When using WSO2 API Manager as the control plane with Choreo Connect, it can be configured to validate the API Subscriptions. For this, the same API should be published in both API Manager and Choreo Connect, and a valid access token (JWT or Reference token) should be obtained by subscribing to the API via an Application. Choreo Connect is capable of validating subscriptions only for the configured tenant (One tenant per Choreo Connect basis).
```

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/3568/commits/0bcdd999288f332d0bd6308bff36e8f7648f4e8c